pkg/cloudevents/transport/http/message.go

```diff
@@ -35,7 +35,10 @@ func (m Message) CloudEventsVersion() string {
 	if m.Header != nil {
 		// Try headers first.
 		// v0.1, cased from the spec
-		if v := m.Header["CE-CloudEventsVersion"]; len(v) == 1 {
```

Comment:
why change this?

Reply:
go vet is checking for incorrect capitalization of string literals used as HTTP headers. If you put the value in a variable it shuts up.